### ISSUE TITLE
Introduce BuiltinProperty model element

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soundasleep/iaml#247]
```
What new or enhanced feature are you proposing?
Once issue 245 is complete, existing PrimitiveConditions should be merged into 
a new model element BuiltinProperty, with a property attribute : property { 
VISIBLE, CAN_SAVE, ... }

What goal would this enhancement help you achieve?
This is a recognised sub-goal of issue 234: Implement metamodel changes as 
discussed in metamodel review
```

Original issue reported on code.google.com by `soundasleep` on 28 Apr 2011 at 1:07
- Blocking: #234, #232
- Blocked on: #245
